src/migration/1697544817927-Invite.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Invite1697544817927 implements MigrationInterface {
+  name = 'Invite1697544817927';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "invite" ("token" text NOT NULL, "campaign" text NOT NULL, "userId" character varying(36) NOT NULL, "limit" integer NOT NULL DEFAULT '5', "count" integer NOT NULL DEFAULT '0', CONSTRAINT "PK_83dbe83cb33c3e8468c8045ea7c" PRIMARY KEY ("token"))`,
+    );
+    await queryRunner.query(
+      `CREATE UNIQUE INDEX "IDX_e31046d3d8b9d04432ccdaaf09" ON "invite" ("userId", "campaign") `,
```

Comment:
Try and add the index in type, it should generate it based on that.

Reply:
Ah missed it on top level I think, missed the regular one.
I do prefer them named myself, but it's maybe personal preference? 

EG: `@Index('IDX_content_image_used_by', ['usedByType', 'usedById'])`

Reply:
Yes named ones are better I agree

Reply:
I prefer them from debugging and logging point of view, easier to identify.
Just bit more explicit to what it indexes for.

Reply:
Yeah, from debugging point of vie we do get a error detail of which column(s) is in violation, which is the same as column would be named. 

But let's have it consistent and enforce naming indexes in guidelines/reviews then.

Reply:
I went ahead with adding a named index here 🙏  but I agree w/ @capJavert , since there are examples of existing indexes which are not named in the project, if this is the way we want to move forward we should start enforcing it everywhere

Reply:
Also, there is an ongoing issue with typeorm where it deletes and reintroduces the same index but with a different name from before when generating migrations (the reason why there are a lot of unrelated lines). Having it named will avoid the issue.